Firestore/core/test/firebase/firestore/core/query_test.cc

```diff
@@ -72,7 +71,7 @@ TEST(QueryTest, Constructor) {
 
 TEST(QueryTest, OrderBy) {
   auto query =
-      Query("rooms/Firestore/messages")
+      testutil::Query("rooms/Firestore/messages")
```

Comment:
Re. the discussion on the name -- I'm sort of inclined to rename the helper method to avoid the issue. I don't feel strongly about this, though. Some suggestions: `MakeQuery`? `TestQuery`? `query`? (capitalization) `make_query`? `ParseQuery`? `Test<Query>`/`Make<Query>`/etc.?

Reply:
I don't feel super strongly about it either, but I'm also not super excited about any of those options. It bugs me that it's an outlier among the rest of the methods, and it's only a few of the tests in `core` that actually see the conflict, but renaming the method affects all the tests.

Reply:
You could also create a local helper function in the affected tests that delegates to the troublesome `Query` and then use it within the file. It doesn't fully solve the consistency problem, though.

Barring that, I find qualifying the test function with the namespace to be a good approach.

Reply:
I'm going to stick with this for now then. Let's revisit if there are more outliers.